src/axom/sidre/tests/sidre_datastore_unit.cpp

```diff
@@ -277,7 +277,7 @@ TEST(sidre_datastore, create_destroy_buffers_views)
 int psrand(int min, int max)
 {
   // Returns a pseudorandom int in [min, max].  Note the closed interval.
-  int range = max - min + 1;
+  long range = max - min + 1;
```

Comment:
Long is still 32-bit on Windows. Maybe "long long" or std::int64_t would reflect the intent of the change better.

Reply:
I agree about using `std::int64_t`